scripts/form-utils.js

```diff
@@ -114,22 +103,18 @@ function postToEmailServer(params, successCallback, errorCallback) {
     xhr.timeout = 5000;
     xhr.ontimeout = errorCallback;
     xhr.responseType = 'json';
-
-    if (params.newsletters) {
-      xhr.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
-      xhr.send(serialize(params));
-    } else {
-      xhr.setRequestHeader("Content-type", "application/json");
-      xhr.send(JSON.stringify(params));
-    }
+    xhr.setRequestHeader("Content-type", "application/json");
+    xhr.send(JSON.stringify(params));
 }
 
 function serialize(params) {
+    const name = encodeURIComponent(params.name);
+    const affiliation = encodeURIComponent(params.affiliation);
+    const title = encodeURIComponent(params.title);
     const email = encodeURIComponent(params.email);
-    const newsletters = encodeURIComponent(params.newsletters);
-    const sourceUrl = encodeURIComponent("https://future.mozilla.org");
+    const sourceUrl = encodeURIComponent("https://open.mozilla.org/letter/");
```

Comment:
Should I be doing anything with the source URL?

Reply:
I'm not sure we need it, I can remove